conf/distro/acrn-demo-uos-rt.conf

```diff
@@ -0,0 +1,15 @@
+require conf/distro/include/acrn-demo.inc
+
+DISTRO .= "-uos"
+DISTRO_NAME += "(UOS)"
+
+PREFERRED_PROVIDER_virtual/kernel ?= "linux-intel-rt-acrn-uos"
+
+# UOS images are always ext4
+IMAGE_FSTYPES = "wic ext4"
+
+# Ensure images typically include networkd-config so that networking is
+# configured.
+MACHINE_ESSENTIAL_EXTRA_RRECOMMENDS += "networkd-config"
+
+APPEND += " clocksource=tsc tsc=reliable x2apic_phys processor.max_cstate=0 intel_idle.max_cstate=0 intel_pstate=disable mce=ignore_ce audit=0 isolcpus=nohz,domain,1 nohz_full=1 rcu_nocbs=1 nosoftlockup idle=poll irqaffinity=0"
```

Comment:
Can we keep these command line parameters for for both 'not preempt-RT' and 'Preempt-RT' kernels along with i915 flags ?

Reply:
We can use "PREFERRED_PROVIDER_virtual/kernel ?= "linux-intel-rt-acrn-uos" or ="linux-intel-acrn-uos" to distinguish RT or no RT image, is that right ?

Reply:
It is true, but I do not like to have multiple distros. Can we add something like this in acrn-demo-uos.conf: 

APPEND += "${@bb.utils.contains('LINUX_KERNEL_TYPE', 'preempt-rt', ' clocksource=tsc x2apic_phys processor.max_cstate=0 intel_idle.max_cstate=0 intel_pstate=disable mce=ignore_ce audit=0 isolcpus=nohz,domain,1 nohz_full=1 rcu_nocbs=1 nosoftlockup idle=poll irqaffinity=0', '', d)}"

Need to test this.

Reply:
it would be nice to have a separate variable for rt specific APPEND so that it provide a default APPEND value and allow to override this part with little effort. (refer to LINUX_GVT_APPEND in acrn-demo-sos)

looks like this: 
LINUX_RT_APPEND ?= ${@bb.utils.contains('LINUX_KERNEL_TYPE', 'preempt-rt', ' clocksource=tsc x2apic_phys ............ irqaffinity=0', '', d)}"
APPEND += "${LINUX_RT_APPEND}"

Reply:
Thanks @cheeyanglee ,  @gaojunhao0504 please follow this to change the code, thanks